newdle/notifications.py

```diff
@@ -48,8 +54,11 @@ def notify_newdle_creator(
 
 
 def send_emails(emails):
-    with get_connection() as conn:
-        return conn.send_messages(emails)
+    try:
+        with get_connection() as conn:
+            return conn.send_messages(emails)
+    except ConnectionRefusedError:
```

Comment:
I don't really like the idea of silently failing when sending emails fails...

Reply:
What would the alternative be? The reason I decided to silence emails is that answering a newdle results in a failure if, for example, SMTP fails. Which is a misconception since the answering actually succeeds.

Reply:
since we don't have any background task for sending emails and I don't want to add this complexity just for that: at least when creating a newdle or notifying people about the final date I'd rather fail and have the user try submitting again (which may work in case of a temporary error). I guess for answering you could argue that losing a notification isn't a big deal.